gnovm/pkg/gnolang/machine.go

```diff
@@ -75,6 +75,7 @@ type MachineOptions struct {
 	Alloc         *Allocator // or see MaxAllocBytes.
 	MaxAllocBytes int64      // or 0 for no limit.
 	MaxCycles     int64      // or 0 for no limit.
+	IsTest        bool
```

Comment:
wdyt if instead of adding `IsTest`, we made `MaxAllocBytes` and `MaxCycles` have default values if `== 0`, and be unlimited if `< 0`?